src/app/pages/system/ca/ca-add/ca-add.component.ts

```diff
@@ -312,6 +314,12 @@ export class CertificateAuthorityAddComponent {
         }
       }
     })
+
+    // Live error check on Identifier (name) field
+    entity.formGroup.controls['name'].statusChanges.subscribe((res) => {
+      res === 'INVALID' ? _.find(this.fieldConfig).hasErrors = true : _.find(this.fieldConfig).hasErrors = false;
```

Comment:
an empty value is also invalid for this field, but the error message is not correct.